lib/v2/dnaindia/category.js

```diff
@@ -0,0 +1,67 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+const { error } = require('winston');
+
+module.exports = async (ctx) => {
+    const { category, topic } = ctx.params;
+    const baseUrl = 'https://www.dnaindia.com';
+    let route;
+    if (category) {
+        route = `/${category}`;
+    } else if (topic) {
+        route = `/topic/${topic}`;
+    } else {
+        error('Invalid URL');
+    }
+    const { data: response } = await got(`${baseUrl}${route}`);
+    const $ = cheerio.load(response);
+
+    const listItems = $('div.col-lg-6 div.list-news')
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            const a = item.find('div.explainer-subtext a');
+            return {
+                title: a.text(),
+                link: `${baseUrl}${a.attr('href')}`,
+            };
+        });
+
+    const items = await Promise.all(
+        listItems.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+                item.itunes_item_image = $('div.article-img img').attr('src');
+                item.category = $('div.tags ul li')
+                    .toArray()
+                    .map((item) => $(item).find('a').text())
+                    .join(', ');
```

Comment:
```suggestion
                    .map((item) => $(item).find('a').text());
```
`item.category` accepts an array of stringg